> retest this please

@st4rl3ss : look at the failure logs in the[ tests tab](https://bbpcode.epfl.ch/ci/blue/organizations/jenkins/hpc.spack-deployment.pull-requests/detail/hpc.spack-deployment.pull-requests/2421/tests) where we have:

```
-- Configuring done
CMake Error: install(EXPORT "MVDToolTargets" ...) includes target "MVDTool" which requires target "Sonata" that is not in the export set.
-- Generating done
CMake Generate step failed. Build files cannot be regenerated correctly.
```

May be this is something relevant error?

--- a/var/spack/repos/builtin/packages/mvdtool/package.py
+++ b/var/spack/repos/builtin/packages/mvdtool/package.py
@@ -16,6 +16,9 @@ class Mvdtool(CMakePackage):
     git      = "https://github.com/BlueBrain/MVDTool.git"
 
     version('develop', get_full_repo=False, submodules=True)
+    version('2.4.2', tag='v2.4.2', get_full_repo=False)
+    # Version 2.4.1 might have some issue in installing via Spack
+    # Please use a more recent version.
     version('2.4.1', tag='v2.4.1', get_full_repo=False)
     version('2.4.0', tag='v2.4.0', get_full_repo=False)
     version('2.3.6', tag='v2.3.6', get_full_repo=False)

--- a/var/spack/repos/builtin/packages/mvdtool/package.py
+++ b/var/spack/repos/builtin/packages/mvdtool/package.py
@@ -17,9 +17,6 @@ class Mvdtool(CMakePackage):
 
     version('develop', get_full_repo=False, submodules=True)
     version('2.4.2', tag='v2.4.2', get_full_repo=False)
-    # Version 2.4.1 might have some issue in installing via Spack
-    # Please use a more recent version.
-    version('2.4.1', tag='v2.4.1', get_full_repo=False)
     version('2.4.0', tag='v2.4.0', get_full_repo=False)
     version('2.3.6', tag='v2.3.6', get_full_repo=False)
     version('2.3.5', tag='v2.3.5', get_full_repo=False)

--- a/var/spack/repos/builtin/packages/py-mvdtool/package.py
+++ b/var/spack/repos/builtin/packages/py-mvdtool/package.py
@@ -16,9 +16,6 @@ class PyMvdtool(PythonPackage):
 
     version('develop', branch='master', submodules=True, get_full_repo=False)
     version('2.4.2', tag='v2.4.2', submodules=True, get_full_repo=False)
-    # Version 2.4.1 might have some issue in installing via Spack
-    # Please use a more recent version.
-    version('2.4.1', tag='v2.4.1', submodules=True, get_full_repo=False)
     version('2.4.0', tag='v2.4.0', submodules=True, get_full_repo=False)
     version('2.3.6', tag='v2.3.6', submodules=True, get_full_repo=False)
     version('2.3.5', tag='v2.3.5', submodules=True, get_full_repo=False)

--- a/var/spack/repos/builtin/packages/py-mvdtool/package.py
+++ b/var/spack/repos/builtin/packages/py-mvdtool/package.py
@@ -15,6 +15,9 @@ class PyMvdtool(PythonPackage):
     git      = "https://github.com/BlueBrain/MVDTool.git"
 
     version('develop', branch='master', submodules=True, get_full_repo=False)
+    version('2.4.2', tag='v2.4.2', submodules=True, get_full_repo=False)
+    # Version 2.4.1 might have some issue in installing via Spack
+    # Please use a more recent version.
     version('2.4.1', tag='v2.4.1', submodules=True, get_full_repo=False)
     version('2.4.0', tag='v2.4.0', submodules=True, get_full_repo=False)
     version('2.3.6', tag='v2.3.6', submodules=True, get_full_repo=False)

